src/components/ContributorTasks/index.tsx

```diff
@@ -6,34 +6,38 @@ import {A} from 'components';
 import {Task} from 'types/github';
 import './ContributorTasks.scss';
 
-interface ComponentProps {
+export interface ContributorTasksProps {
   className?: string;
   tasks: Task[];
 }
 
-const ContributorTasks: FC<ComponentProps> = ({className, tasks}) => {
+const ContributorTasks: FC<ContributorTasksProps> = ({className, tasks}) => {
   const renderRows = () => {
     return tasks.map(({amount_paid, completed_date, issue_id, repository, title}: Task, index) => (
-      <tr key={index}>
-        <td className="ContributorTasks__task-title" title={title}>
+      <tr key={index} data-testid="ContributorTasks--row">
```

Comment:
lets change this name to `ContributorTasks__row` 

( __ denotes an element, while -- denotes a modifier)